glossary.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Glossary 
+description: A collection of technical terminology commonly used in the Astro documentation and community.
+i18nReady: false
+---
+
+This glossary describes common terminology used in the Astro documentation and the Astro community.
+
+{ /* // Uncomment this section when we have more terms to add to the glossary.
+It is separated into Astro-specific terms and general web development terms.
+## Astro-Specific Terms 
+
+Some future guidelines for this:
+- Astro-Specific: Content Collections, ViewTransitions, Astro Component
+- General: Editor, LSP, Front-End Library, Framework, Metaframework, Framework Component
+*/}
+
+### Component
+
+Astro components are the basic building blocks of any Astro project. They are HTML-only templating components with no client-side runtime. You can spot an Astro component by its file extension: `.astro`. Read more about components in the [Astro Components](https://docs.astro.build/en/core-concepts/astro-components/) page.
```

Comment:
```suggestion
Astro components are the basic building blocks of any Astro project. They are HTML-only templating components with no client-side runtime. You can spot an Astro component by its file extension: `.astro`. 

Source: [Astro Components](https://docs.astro.build/en/core-concepts/astro-components/)
```
What would you think about a format more like this, that avoids needing an extra sentence? Simply referencing the most appropriate page/section? Play around with the format (maybe even linking inside the main text like:

```suggestion
[Astro Components](https://docs.astro.build/en/core-concepts/astro-components/) are the basic building blocks of any Astro project. They are HTML-only templating components with no client-side runtime. You can spot an Astro component by its file extension: `.astro`. 
```
... but I think the fewer non-essential sentences here, the better?

Reply:
What about a mix of both styles? For "Components" I like the second one (link within the main text) a lot more, but for some terms the page title might not appear in the text.